airbyte-integrations/connectors/source-github/fixtures/run.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
 
-read -p "Enter your repository ssh link: " ssh_url
-./scripts/init_repository.sh $ssh_url
-read -p "Enter your api_key: " api_key
-read -p "Enter your repository name: airbyte/airbyte-main: " repo
+read -r -p "Enter your repository ssh link: " ssh_url
+./scripts/init_repository.sh "$ssh_url"
+read -r -p "Enter your api_key: " api_key
+read -r -p "Enter your repository name: airbyte/airbyte-main: " repo
```

Comment:
-r ensures that input backslash characters are interpreted as a character in the input string, and not as a regex escape character

Reply:
Same as previously, could you add the comment to the code directly?